src/ducks/resource.js

```diff
@@ -151,11 +152,17 @@ function selectTranslation(original, translations, type, language) {
 function updateTranslation(original, translation, updatedField, value) {
   return (dispatch) => {
     const strings = {};
+    const string_versions = {};
+
     Object.keys(original.strings).forEach((field) => {
       strings[field] = translation.strings[field] || '';
+      string_versions[field] = translation.string_versions[field];
```

Comment:
Should a default be set here if `translation.string_versions[field]` is undefined?

Reply:
On the Panoptes side it's fine to leave it null.

Reply:
If it's a new field that's present in the original but not the translation, would I be right in saying the translation version should be `original.string_versions[field]`?

Reply:
Only if it's actually been edited. The goal is to not change the string_versions except for the key of the field being edited.

Reply:
That makes sense. Thanks!